### PR TITLE
Redesign java

### PR DESCRIPTION
**Pull request by ekcasey**

### CODE REVIEW

File: content/docs/buildpacks/configuration.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+
+If a given [language family buildpack][language family buildpacks] does not contain the Procfile Buildpack it can be explicitly appended at runtime.
+
+**Example**: A Hello World Procfile
+
+The following adds a process with `type` equal to `hello` and makes it the default process.
+{{< code/copyable >}}
+echo "hello: echo hello world" > nodejs/yarn
+pack build samples/nodejs \
+  --path nodejs/yarn \
```
Comment (kvedurmu) [inline content/docs/buildpacks/configuration.md:181]:
```suggestion
  --path nodejs/yarn/Procfile \
```

File: content/docs/buildpacks/configuration.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+  --buildpack paketo-buildpacks/nodejs \
+  --buildpack paketo-buildpacks/procfile \
```
Comment (kvedurmu) [inline content/docs/buildpacks/configuration.md:183]: When I tried this as specified, I see the error `ERROR: failed to build: processing order: unable to resolve version: multiple versions of paketo-buildpacks/procfile - must specify an explicit version`. Should we instead just reference the GCR path for procfile?

**Review by kvedurmu — CHANGES_REQUESTED** [review]
Overall looks awesome! Really like the Configuring Paketo Buildpacks page. Perhaps we can figure out a way to break this up better in the future, but this is fine for now. 

It also looks like the addition of the new subnav "language family buildpack" page breaks the Logo [links on the home page](https://github.com/paketo-buildpacks/paketo-website/blob/redesign/layouts/partials/hero.html#L17).

File: content/docs/buildpacks/configuration.md (hunk 2 above)
Comment (ekcasey) [reply content/docs/buildpacks/configuration.md:183]: Yeah good catch, I am seeing the same behavior. This actually seems like a `pack` bug to me, given there is only one version of `paketo-buildpacks/procfile` on the builder image. I'll switch to using the image reference for now and file a bug with `pack`.

Comment (ekcasey) [reply content/docs/buildpacks/configuration.md:183]: https://github.com/buildpacks/pack/issues/858

File: content/docs/buildpacks/configuration.md (hunk 1 above)
Comment (ekcasey) [reply content/docs/buildpacks/configuration.md:181]: Did you mean this line instead? https://github.com/paketo-buildpacks/paketo-website/pull/48/commits/301240792c6cd0b99fb4ad76c2a91ae22a6b1cf7#diff-011af7d1796feaa60e53963bb34a6c24R179

Comment (kvedurmu) [reply content/docs/buildpacks/configuration.md:181]: Oops. Yeah. Your latest change looks good to me.

**Review by kvedurmu — APPROVED** [review]
Looks great!

**Pull request merged**